src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1565,7 +1565,7 @@ internal enum ErrorCode
         ERR_NewBoundWithUnmanaged = 8375,
         ERR_UnmanagedConstraintMustBeFirst = 8376,
         ERR_UnmanagedConstraintNotSatisfied = 8377,
-        ERR_UnmanagedConstraintWithLocalFunctions = 8378,
+        // not used/claimable = 8378,
```

Comment:
>// not used/claimable = 8378, [](start = 8, length = 29)

We are pretty much done with 7.3, consider removing the comment and adjusting values for the ids below.

Reply:
If there is no objection, I'd rather comment it and reuse it in a later feature (probably a coming bug fix in the following couple of weeks) than changing many more unrelated files to this PR to make sure error string comments in tests and such are correct.

---
In reply to: [177223314](https://github.com/dotnet/roslyn/pull/25680#discussion_r177223314) [](ancestors = 177223314)